sdk/attestation/azure-security-attestation/test/ut/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
+#cspell: words tpmattestation
```

Comment:
This could also take the form of an [override](https://microsoft.sharepoint.com/:w:/r/teams/BoomTeam/_layouts/15/Doc.aspx?sourcedoc=%7BEB3F9A10-08D5-4450-943B-03DDFECA7F60%7D&file=Signtool_Disabling_Page_Hashes.doc&action=default&mobileredirect=true&DefaultItemOpen=1&share=IQEQmj_r1QhQRJQ7A93-yn9gASsu9yT39t8TJiB9W1jHIok) 

Something like: 

```
    "overrides": [
      {
        "filename": "sdk/attestation/**",
        "words": [
          "tpmattestation"
        ]
      },
    ]
```

The advantage of overrides is that we're not coupling our source files to cspell.

Reply:
DOne, great suggestion.

Reply:
This comment should be removed :)